pyresample/bucket/__init__.py

```diff
@@ -202,20 +202,29 @@ def _get_indices(self):
         target_shape = self.target_area.shape
         self.idxs = self.y_idxs * target_shape[1] + self.x_idxs
 
-    def get_sum(self, data, skipna=True):
+    def get_sum(self, data, fill_value=np.nan, skipna=True, empty_bucket_value=0):
         """Calculate sums for each bin with drop-in-a-bucket resampling.
 
         Parameters
         ----------
         data : Numpy or Dask array
             Data to be binned and summed.
+        fill_value : float
+            Fill value to mark missing/invalid values in the input data.
```

Comment:
The previous wording suggest to me on first reading that the `fill_value` was used to write/assign to some array, but if I'm understanding it correctly it is already existing in the input data so I thought maybe this could be less confusing:

```suggestion
            Fill value of the input data marking missing/invalid values.
```

Reply:
indeed, fixed as proposed.